homeassistant/components/home_connect/select.py

```diff
@@ -66,18 +66,17 @@ def __init__(
         desc: SelectEntityDescription,
     ) -> None:
         """Initialize the entity."""
-        super().__init__(
-            coordinator,
-            appliance,
-            desc,
-        )
         self._attr_options = [
             PROGRAMS_TRANSLATION_KEYS_MAP[program.key]
             for program in appliance.programs
             if program.key != ProgramKey.UNKNOWN
         ]
+        super().__init__(
```

Comment:
Why do we move this?

Reply:
I think that it was a change that I made while the migration to aiohomeconnect was WIP, and it was because initializing `HomeConnectEntity` triggered `async_write_ha_state`, and because options weren't set, it raised an exception.
Any way, it doesn't happen any more, so I'm reverting that change